src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Application {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[new Random().nextInt(6)];
+        for (int i = 0; i < figures.length; i++) {
```

Comment:
inside the for loop, you should check in if-else, whether you need to create the random figure (first half), or the default one (second half). For this, you need to access the figures.length, which better be constant, than randomized int (random can return 0 or 1, which won't let us test anything properly).